app/src/Pages/Home/Home.js

```diff
@@ -316,64 +339,113 @@ class HomePage extends React.Component {
         >
           {!!mapData.features.length && (
             <>
-              <BaseMap
-                updateViewport={false}
-                initialZoom={11}
-                useContainerHeight
+              <div
```

Comment:
General comment about this file, it could be nice to pull out pieces of the component as sub-components so that this component is more easily readable though it's not essential. The longer or more complex this file gets, the harder it will be for a future random engineer to jump in and improve it. Perhaps a wrapper component that maintains the state and updates with a form component that's been busted out which updates the wrapper component's state, and visual sub-components, such as the map, updating based on those state changes.